lib/utils/fs.go

```diff
@@ -94,11 +94,19 @@ func NormalizePath(path string, evaluateSymlinks bool) (string, error) {
 	if err != nil {
 		return "", trace.ConvertSystemError(err)
 	}
+
 	if evaluateSymlinks {
 		s, err = filepath.EvalSymlinks(s)
 		if err != nil {
 			return "", trace.ConvertSystemError(err)
 		}
+
+		// If the symlink is relative, filepath.EvalSymlinks returns a path relative to the current dir.
+		// Resolve it to an absolute path to stay true to the godoc.
+		s, err = filepath.Abs(s)
```

Comment:
Could we add test coverage for this?

Reply:
I misunderstood how [`filepath.EvalSymlinks`](https://pkg.go.dev/path/filepath#EvalSymlinks) works.

> EvalSymlinks returns the path name after the evaluation of any symbolic links. If path is relative the result will be relative to the current directory, unless one of the components is an absolute symbolic link. EvalSymlinks calls [Clean](https://pkg.go.dev/path/filepath#Clean) on the result.

Specifically, "If path is relative" is about the path passed to `EvalSymlinks`, not the path of the symlink. I'll remove the changes to `lib/utils.NormalizePath`.